outcomes/firefox_desktop/firefox_suggest.toml

```
@@ -0,0 +1,70 @@
+[metrics.urlbar_search_count]
```

whoops, I missed this earlier. You'll need to have a `friendly_name` and a description that Experimenter will show.
```suggestion
friendly_name = "Firefox Suggest"
description = "Usage and engagement metrics for Firefox Suggest."

[metrics.urlbar_search_count]
```